pkg/test/resourcefixture/testdata/basic/alloydb/v1beta1/alloydbcluster/fullalloydbcluster/_generated_object_fullalloydbcluster.golden.yaml

```diff
@@ -41,14 +41,13 @@ spec:
   clusterType: PRIMARY
   continuousBackupConfig:
     enabled: false
-    recoveryWindowDays: 14
```

Comment:
Do you know why this is changed?

Reply:
I think it's because we weren't running this test previously (because it depends on KMSCryptoKey).

It's a gap in the mockgcp for AlloydbCluster.  It should default recoveryWindowDays to 14 I think.

The tricky thing is ... do we want to do this in this PR?  If we do, the PR becomes even bigger.  I do think once we get support for KMSCryptoKey in, we will want to do a `record-gcp` followed by a `compare-mock` on the AlloyDB fixtures, to fix up any loose ends in mockgcp here.

Reply:
I see. That makes sense.

Agree to complete the AlloyDb later once the KMSCryptoKey is in.